media-video/gaupol/gaupol-1.14.1-r1.ebuild

```
@@ -0,0 +1,81 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit distutils-r1 virtualx xdg-utils
+
+DESCRIPTION="A subtitle editor for text-based subtitles"
+HOMEPAGE="https://otsaloma.io/gaupol/ https://github.com/otsaloma/gaupol/"
+SRC_URI="https://github.com/otsaloma/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="+player spell"
+
+RDEPEND="
+	app-text/iso-codes
+	dev-python/chardet[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	x11-libs/gtk+:3[introspection]
+	player-integrated? (
+		>=media-libs/gstreamer-1.6
+		media-libs/gst-plugins-bad
+		media-libs/gst-plugins-base[pango]
+		media-libs/gst-plugins-good
+		media-plugins/gst-plugins-gtk
+		media-plugins/gst-plugins-libav
+	)
+	spell? ( app-text/gspell[introspection] )
+"
+BDEPEND="
+	sys-devel/gettext
+	test? (
+		app-dicts/myspell-en
+		app-text/enchant[hunspell]
+		app-text/gspell[introspection]
+	)
+"
+
+distutils_enable_tests pytest
+
+DOCS=( AUTHORS.md NEWS.md README.md README.aeidon.md )
+
+PATCHES=(
+	"${FILESDIR}/${PN}-1.12-fix-prefix.patch"
+)
+
+python_test() {
+	virtx epytest
+}
+
+pkg_postinst() {
+	xdg_desktop_database_update
+	xdg_icon_cache_update
+
+	optfeature_header "Gaupol supports those external players"
+	optfeature "MPlayer" media-video/mplayer
+	optfeature "MPV" media-video/mpv
+	optfeature "VLC" media-video/VLC
```

```suggestion
	optfeature "external player support" media-video/mplayer media-video/mpv media-video/VLC
```

Even so id be mixed on the necessity of this.